game/screen_paths.cc

```diff
@@ -101,11 +101,10 @@ void ScreenPaths::generateMenuFromPath(fs::path path) {
 				std::clog << "screen_paths/notice: Ignoring hidden folder: ." << p.c_str() << std::endl;
 				continue;
 			} else {
-				m_menu.add(MenuOption(p.c_str(),_("Open folder")).call([this, p](){
-					generateMenuFromPath(p);
+				m_menu.add(MenuOption(boost::lexical_cast<std::string>(p),_("Open folder")).call([this, p](){ generateMenuFromPath(p);
```

Comment:
ah that was the problem? but why does linux just take it without cast and windows not?